repos.d/rpm/unitedrpms.yaml

```diff
@@ -52,18 +32,18 @@
     - name: repo
       fetcher: RepodataFetcher
       parser: RepodataParser
-      url: https://downloads.sourceforge.net/project/unitedrpms/28/srpm/
+      url: https://downloads.sourceforge.net/project/unitedrpms/29/srpm/
       subrepo: release
   repolinks:
     - desc: UnitedRPMs home
       url: https://unitedrpms.github.io/
     - desc: GitHub organization
       url: https://github.com/UnitedRPMs
   tags: [ all, production, unitedrpms, rpm ]
-
-- name: unitedrpms_29
+  
+  - name: unitedrpms_30
```

Comment:
Here. Extra space is visible on the diff with naked eye, diagnostics point to this line as well.

Reply:
done!